Monika After Story/game/screens.rpy

```diff
@@ -3,34 +3,30 @@ init 100 python:
     layout.UNSTABLE = store.mas_layout.UNSTABLE
 
 init -1 python:
-    layout.QUIT_YES = "Please don't close the game on me!"
-    layout.QUIT_NO = "Thank you, [player]!\nLet's spend more time together~"
+    layout.QUIT_YES = _("Please don't close the game on me!")
+    layout.QUIT_NO = _("Thank you, [player]!\nLet's spend more time together~")
 
     # tooltips
-    layout.MAS_TT_SENS_MODE = (
-        "Sensitive mode removes content that may be disturbing, offensive, "
-        " or considered tasteless."
+    layout.MAS_TT_SENS_MODE = _(
+        "Sensitive mode removes content that may be disturbing, offensive, or considered tasteless."
     )
-    layout.MAS_TT_UNSTABLE = (
-        "Unstable mode downloads updates from the experimental unstable "
-        "branch of development. It is HIGHLY recommended to make a backup "
-        "of your persistents before enabling this mode."
+    layout.MAS_TT_UNSTABLE = _(
+        "Unstable mode downloads updates from the experimental unstable branch of development. It is HIGHLY recommended to make a backup of your persistents before enabling this mode."
     )
-    layout.MAS_TT_REPEAT = (
+    layout.MAS_TT_REPEAT = _(
         "Enable this to let Monika repeat topics that you have already seen."
     )
-    layout.MAS_TT_NOTIF = (
+    layout.MAS_TT_NOTIF = _(
         "Enabling this will let Monika use your system's notifications and check if MAS is your active window "
     )
-    layout.MAS_TT_NOTIF_SOUND = (
+    layout.MAS_TT_NOTIF_SOUND = _(
         "If enabled, a custom notification sound will play for Monika's notifications "
     )
-    layout.MAS_TT_G_NOTIF = (
+    layout.MAS_TT_G_NOTIF = _(
         "Enables notifications for the selected group."
     )
-    layout.MAS_TT_ACTV_WND = (
-        "Enabling this will allow Monika to see your active window "
-        "and offer some comments based on what you're doing."
+    layout.MAS_TT_ACTV_WND = _(
+        "Enabling this will allow Monika to see your active window and offer some comments based on what you're doing."
```

Comment:
long string lines are split to keep the character line limit to 80-120 characters. please split these lines like they were before.